components/blobserve/leeway.Dockerfile

```diff
@@ -2,7 +2,7 @@
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License.AGPL.txt in the project root for license information.
 
-FROM alpine:3.16
+FROM cgr.dev/chainguard/wolfi-base:latest
```

Comment:
Why use `latest` rather than versioned?

Reply:
@utam0k there is only one tag https://edu.chainguard.dev/chainguard/chainguard-images/reference/wolfi-base/overview/

Reply:
On this page, they make it very clear that the image is experimental. Do you believe it is reliable enough to have our production images based on it?